### DISCUSSION TITLE
Feature Request: Grafana 8.0 Ruler API not accesible on same URL as Cortex datasource

### DISCUSSION THREAD

**Discussion by kubicgruenfeld** [grafana/grafana#43662] [Alerting]
**What would you like to be added**:

The possiblity to configure the ruler API on an other url in Prometheus datasources since they must not be the same.

**Why is this needed**:

To use ngalert e.g. in Grafana deployments in k8s clusters with Cortex the url of the Prometheus datasources would most likely use something like `http://query-frontend/...` and the ruler would be reachable with an other service like `http://ruler/...`

**Comment by domasx2** [comment]
Perhaps we could add a "ruler" datasource that could be linked to a existing prometheus datasource :thinking:

**Comment by domasx2** [comment]
Current plan is instead of creating a new ruler datasource, add optional alternative HTTP config for ruler to Prometheus / Loki datasources

**Comment by grobinson-grafana** [comment]
We are closing this as it is a duplicate of https://github.com/grafana/grafana/issues/45825.
